AWSCoreTests/AWSTestUtility.h

```diff
@@ -22,6 +22,12 @@ FOUNDATION_EXPORT NSString *const AWSTestUtilityCognitoIdentityServiceKey;
 
 @interface AWSTestUtility : NSObject
 
++ (void)setupSessionCredentialsProvider;
++ (NSDictionary *)getIntegrationTestConfigurationFor:(NSString *) packageId;
```

Comment:
nit: ObjectiveC naming is a bit more verbose than Swift. Change to:

```objective-c
+ (NSDictionary *)getIntegrationTestConfigurationForPackageId:(NSString *) packageId;
```

Reply:
Done